schematic/schemas/data_model_jsonld.py

```
@@ -0,0 +1,284 @@
+                                                node_2_id not in template[rel_vals['jsonld_key']]):
+                                                template[rel_vals['jsonld_key']].append(node_2_id)
+                                            else:
+                                                template[rel_vals['jsonld_key']] == node_2
+            # Fill in node value information
+            else:               
+                node_label = rel_vals['node_label']
+                
+                # Get recorded info for current node, and the attribute type
+                node_info = nx.get_node_attributes(self.graph, node_label)[node]
+                
+                # Add this information to the template
+                template[rel_vals['jsonld_key']] =  node_info
+        
+        # Clean up template
+        template = self.clean_template(template=template,
+                                       data_model_relationships=data_model_relationships,
+                                       )
+        # Reorder lists based on weights:
+        template = self.reorder_template_entries(template=template,)
+
+        # Add contexts to certain values
+        template = self.add_contexts_to_entries(template=template,)
+
+        return template
+
+    def add_contexts_to_entries(self, template:dict) -> dict:
+        """
+        Args:
+            template, dict: JSONLD template that has been filled up to the current node, with information
+        Returns:
+            template, dict: JSONLD template where contexts have been added back to certain values.
+        Note: This will likely need to be modified when Contexts are truly added to the model
+        """
+        for jsonld_key, entry in template.items():
+            try:
+                # Retrieve the relationships key using the jsonld_key
+                key= [k for k, v in self.rel_dict.items() if jsonld_key == v['jsonld_key']][0]
+            except:
```

I don't think you want to catch all exceptions here?